docs/content/sre/tenant-operator/usecases/argocd.md

```diff
@@ -180,3 +180,38 @@ spec:
     kind: Environment
   ...
 ```
+
+### Override NamespaceResourceBlacklist and ClusterResourceWhitelist per Tenant
+
+Bill now wants a specific tenant to override the `namespaceResourceBlacklist` and `clusterResourceWhitelist` set via Integration Config. Bill will specify these in `argoCD.appProjects` section of Tenant spec. 
+
```

Comment:
modify the sentence/heading to make sure the reader knows that only one of the two can be overriden

Reply:
simplest way is to use "and/or"